### ISSUE TITLE
Failure when starting dojot with docker-compose

### ISSUE THREAD

**Issue by rascaraficci** [dojot/dojot#874]
After run:

``` sh
docker-compose up -d
```
All dojot's services seeming to be running, but the API wasn't responding. Looking at the log of the apigw, there were some error messages:

```
Missing dependencies for pepkong 0.0-1:
   json4lua ~> 0.9.30-1 (not installed)

pepkong 0.0-1 depends on json4lua ~> 0.9.30-1 (not installed)
Installing https://luarocks.org/json4lua-0.9.30-1.src.rock
Do not use 'module' as a build type. Use 'builtin' instead.
json4lua 0.9.30-1 is now installed in /usr/local (license: GPL)

pepkong 0.0-1 is now installed in /usr/local (license: null)

/plugins
prefix directory /usr/local/kong not found, trying to create it
2019/01/18 15:23:36 [warn] postgres database 'kong' is missing migration: (response-ratelimiting) 2015-08-03-132400_init_response_ratelimiting
Error: /usr/local/share/lua/5.1/kong/cmd/start.lua:28: [postgres error] the current database schema does not match this version of Kong. Please run `kong migrations up` to update/initialize the database schema. Be aware that Kong migrations should only run from a single node, and that nodes running migrations concurrently will conflict with each other and might corrupt your database schema!

  Run with --v (verbose) or --vv (debug) for more details
pepkong 0.0-1 is now installed in /usr/local (license: null)

/plugins
``` 

It seems that something bad happened to the database. After stopping and starting all the services, the apigw started without errors.

It would be nice to have some validations to guarantee that if the services are up dojot is operational.
